source/extensions/filters/network/thrift_proxy/router/upstream_request.cc

```diff
@@ -300,6 +300,7 @@ bool UpstreamRequest::onResetStream(ConnectionPool::PoolFailureReason reason) {
 
     // TODO(zuercher): distinguish between these cases where appropriate (particularly timeout)
     if (response_started_) {
+      ENVOY_LOG(warn, "reset downstream connection for a partial response");
```

Comment:
`warn` level may be a bit high or does http use this level?

Reply:
http would send local reply iirc. however, not much warn log in http, so happy to down level to debug

Reply:
warn might not be that bad, i'd be curious to know how much it happens in your prod world.